services/credential-server/src/agent.ts

```diff
@@ -42,12 +44,16 @@ class Agent {
     await signifyReady();
     let bran;
     let issuerBran;
-    const bransFilePath = "../brans.json";
+    const bransFilePath = "../../brans-data/brans.json";
```

Comment:
I think something like `./data` makes sense - as we don't know where people will have cloned this repository - you might have permissions issues.

And `./data` is a generic name in case we have more than just brans to store.